### ISSUE TITLE
Unable to load saved games

### ISSUE THREAD

**Issue by Blzut3** [ECWolfEngine/ECWolf#56]
**[Original report](https://bitbucket.org/ecwolf/ecwolf/issues/56) by Federico Bett (Bitbucket: [gfbett](https://bitbucket.org/gfbett), GitHub: [gfbett](https://github.com/gfbett)).**

----------------------------------------

If a game is saved, the application is exited and opened again the load game option is not available. 

It seems that the games are being saved on "%APPDATA%" folder, but being loaded from the "./" directory

Code Snippet: wl_loadsave.cpp

```
#!c++

void SetupSaveGames()
{
	char title[MAX_SAVENAME+1];

	File saveDirectory("./");
	const TArray<FString> &files = saveDirectory.getFileList();

```
Workaround: Launching the game with "--savedir ." allows the savegames to be loaded

**Comment by Blzut3** [comment]
Well that's embarrassing. Looks like a hotfix will be coming out shortly (kind of want to wait for an answer on the sound card issue mentioned on Doomworld first). Anyhow, fixed in changeset 9b7ea45.

**Comment by Blzut3** [comment]
**Original comment by Federico Bett (Bitbucket: [gfbett](https://bitbucket.org/gfbett), GitHub: [gfbett](https://github.com/gfbett)).**

----------------------------------------

Thanks for the quick fix!
